src/styles/card.scss

```diff
@@ -0,0 +1,94 @@
+.card {
+  max-width: 200px;
+  min-height: 408px;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
```

Comment:
The border color (`#f3f3f3`) should be defined as an SCSS variable. This will make it easier to update the color scheme in the future.